### DISCUSSION TITLE
報告や相談はどの程度のレベルから行えばよいか

### DISCUSSION THREAD

**Discussion by kagomen** [first-contributions-ja/first-contributions-ja.github.io#39] [【運営】相談・報告]
#2 で質問をさせていただいたのですが、この程度のことは相談や共有などせず、独断で決めてしまった方が良いと思いますか❔

**Comment by kazzyfrog** [comment]
これも、色々進め方とか一回決めてみてもいいかもですね！

基本的には、issueとかで質問・ディスカッションするのは全然良いと思ってます
ただ、反応がすぐにこない場合もあるので、独断で決めて進めてしまって、PR出しても良いと思いますよ！

そこで問題があれば、変更を求められると思うので、楽な方で大丈夫です！

**Reply by pss-aileen** [reply]
私も同じくそう思います！
私も結構勝手にやってしまうタイプなので、
返事来なかったらとりあえず作ってコミットなどして
みんなどうですか〜〜〜って呼びかける感じでも良いかと思います！

**Comment by kagomen** [comment]
お二人ともご意見ありがとうございます。

> これも、色々進め方とか一回決めてみてもいいかもですね！

そうですね。
やっていくうちに着地点を掴める気もしますが、最初から決まってると楽なのかなーとは思いました。
ただ、決め事を作るのも、その決め事を確認したり覚えたりするのも労力がいる作業ではあるので、何を決めるかを決めるのも、難しいですね……！

とりあえずあまり怖がらずにPR出すことにも挑戦してみることにします……！
